src/fabmap.cpp

```diff
@@ -51,623 +51,534 @@
 // possibility of such damage.
 //////////////////////////////////////////////////////////////////////////////*/
 
-#include "../include/openfabmap.hpp"
-
-using std::vector;
-using std::list;
-using std::map;
-using std::multiset;
-using std::valarray;
-using cv::Mat;
+#include "../include/fabmap.hpp"
 
 /*
-	Calculate the sum of two log likelihoods
+    Calculate the sum of two log likelihoods
 */
-double logsumexp(double a, double b) {
-	return a > b ? log(1 + exp(b - a)) + a : log(1 + exp(a - b)) + b;
+namespace of2 {
```

Comment:
I'm using it in another (yet uncommitted) file. There might be a more appropriate place for it to live, this was a quick fix.